apps/e2e/cypress/e2e/instruments.cy.ts

```diff
@@ -639,6 +718,64 @@ context('Instrument tests', () => {
       cy.contains(proposal1.title);
     });
 
+    it('Instrument scientist should be able to filter proposals assigned to multiple instruments where assigned as instrument scientist', () => {
+      cy.createProposal({ callId: initialDBData.call.id }).then(
+        ({ createProposal }) => {

```

Comment:
This will require a bit of a refactor if we want to change because it comes from the type that is autogenerated by graphql sdk. So it is defined on the backend in the end and it will change it for all the places where it's used.

Reply:
Oh, I didn't realise that. Don't bother then, it doesn't matter.